demo/psa/prov-verif-e2e/COCLI_README.md

```diff
@@ -0,0 +1,59 @@
+# Provisioning Instructions
+
+Follow below instructions if one wishes to create new CoMID's and use them in creating CoRIM's to provision them in Veraison Verification Service.
```

Comment:
Maybe a link to definitions of `CoMID` and `CoRIM`, in case this this the first part of the docs someone is reading?

Reply:
@setrofim Valid point, I will add a link to main CoRIM Draft.